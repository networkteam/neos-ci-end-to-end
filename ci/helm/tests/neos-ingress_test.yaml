templates:
  - neos-ingress.yaml
tests:
  - it: matches snapshot with review deployment
    set:
      neos:
        ingress:
          hostnames:
            - neos-my-release.k.networkteam.com
          tls:
            - secretName: wildcard-k-networkteam-com
              hosts:
                - neos-my-release.k.networkteam.com

      default:
        ingress:
          annotations:
            nginx.ingress.kubernetes.io/auth-realm: Authentication Required
        annotations:
          app.gitlab.com/env: foo
          app.gitlab.com/app: bar
    release:
      name: my-release
    asserts:
      - matchSnapshot: {}
