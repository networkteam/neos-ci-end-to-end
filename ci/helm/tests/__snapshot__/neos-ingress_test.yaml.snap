matches snapshot with review deployment:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations:
        app.gitlab.com/app: bar
        app.gitlab.com/env: foo
        nginx.ingress.kubernetes.io/auth-realm: Authentication Required
      labels:
        app.kubernetes.io/component: neos
        app.kubernetes.io/instance: my-release
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: neos-ci
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: neos-ci-0.1.0
      name: my-release-neos
    spec:
      ingressClassName: nginx
      rules:
      - host: neos-my-release.k.networkteam.com
        http:
          paths:
          - backend:
              service:
                name: my-release-neos
                port:
                  number: 80
            path: /
            pathType: ImplementationSpecific
      tls:
      - hosts:
        - neos-my-release.k.networkteam.com
        secretName: wildcard-k-networkteam-com
